site/src/pages/AuditPage/AuditPageView.tsx

```diff
@@ -72,53 +78,88 @@ export const AuditPageView: FC<AuditPageViewProps> = ({
+                        <TableRow>
+                          <TableCell colSpan={999}>
+                            <EmptyState message={t("table.emptyPage")} />
+                          </TableCell>
+                        </TableRow>
+                      </Cond>
+                      <Cond>
+                        <TableRow>
+                          <TableCell colSpan={999}>
+                            <EmptyState message={t("table.noLogs")} />
+                          </TableCell>
+                        </TableRow>
+                      </Cond>
+                    </ChooseOne>
                   </Cond>
                   <Cond>
-                    <TableRow>
-                      <TableCell colSpan={999}>
-                        <EmptyState message={t("table.noLogs")} />
-                      </TableCell>
-                    </TableRow>
+                    {auditLogs && (
+                      <Timeline
+                        items={auditLogs}
+                        getDate={(log) => new Date(log.time)}
+                        row={(log) => (
+                          <AuditLogRow key={log.id} auditLog={log} />
+                        )}
+                      />
+                    )}
                   </Cond>
                 </ChooseOne>
-              </Cond>
-              <Cond>
-                {auditLogs && (
-                  <Timeline
-                    items={auditLogs}
-                    getDate={(log) => new Date(log.time)}
-                    row={(log) => <AuditLogRow key={log.id} auditLog={log} />}
-                  />
-                )}
-              </Cond>
-            </ChooseOne>
-          </TableBody>
-        </Table>
-      </TableContainer>
+              </TableBody>
+            </Table>
+          </TableContainer>
+
+          <PaginationWidget numRecords={count} paginationRef={paginationRef} />
+        </Cond>
 
-      <PaginationWidget numRecords={count} paginationRef={paginationRef} />
+        <Cond>
+          <Paywall
+            message="Audit logs"
+            description="Audit Logs allows Auditors to monitor user operations in their deployment. To use this feature, you have to upgrade your account."
+            cta={
+              <Stack direction="row" alignItems="center">
+                <Link
+                  underline="none"
+                  href="https://coder.com/docs/coder-oss/latest/admin/upgrade"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  <Button size="small" startIcon={<ArrowRightAltOutlined />}>
+                    See how to upgrade
+                  </Button>
+                </Link>
+                <Link
+                  underline="none"
+                  href="https://coder.com/docs/coder-oss/latest/admin/audit-logs"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  Read the docs
```

Comment:
We need some translations here, for these links, and also for the paywall message and description.

Reply:
Totally